packages/api/tests/serial/endpoints/test-pdrs.js

```diff
@@ -230,7 +232,8 @@ test('DELETE handles the case where the PDR exists in S3 but not in DynamoDb', a
 
 test('DELETE handles the case where the PDR exists in DynamoDb but not in S3', async (t) => {
   const newPdr = fakePdrFactory('completed');
-  await pdrModel.create(newPdr);
+  const record = await pdrModel.create(newPdr);
+  await indexer.indexPdr(esClient, record, esIndex);
```

Comment:
Should indexing the PDR be necessary now that we added logic to prevent errors being thrown when deleting from ES if it doesn't exist?

Reply:
I think no? But I'll run the tests

Reply:
Success